libcaf_test/caf/test/runner.cpp

```diff
@@ -3,19 +3,18 @@
 // https://github.com/actor-framework/actor-framework/blob/master/LICENSE.
 
 #include "caf/test/runner.hpp"
```

Comment:
This is the main header. So that's one of the few places where we keep the empty line. 🙂

```suggestion
#include "caf/test/runner.hpp"

```